include/gatekeeper_fib.h

```diff
@@ -217,6 +246,7 @@ int add_fib_entry(const char *prefix, const char *gt_ip, const char *gw_ip,
 int del_fib_entry_numerical(
 	struct ip_prefix *prefix_info, struct gk_config *gk_conf);
 int del_fib_entry(const char *ip_prefix, struct gk_config *gk_conf);
+const luaL_Reg *get_fib_dump_reg(void);
```

Comment:
We should replace the prototype of `get_fib_dump_reg()` with the export of `dylib_lua_c_funcs`.